server/src/main/java/org/apache/druid/server/coordinator/rules/Rules.java

```diff
@@ -19,12 +19,15 @@
 
 package org.apache.druid.server.coordinator.rules;
 
+import org.apache.druid.java.util.common.DateTimes;
 import org.joda.time.DateTime;
 import org.joda.time.Interval;
 import org.joda.time.Period;
 
 public class Rules
 {
+  public static final Interval FOREVER_INTERVAL = new Interval(DateTimes.utc(Long.MIN_VALUE), DateTimes.utc(Long.MAX_VALUE));
+
   public static boolean eligibleForLoad(Interval src, Interval target)
```

Comment:
The eligibility methods should probably now use the newly added `getInterval` method. This would simplify the code and would have the added benefit of testing the `getInterval` method through existing unit tests.

Reply:
hmm, perhaps, but I'm not very clear what that would look like. I wanted to scope it down to API validation, so we can revisit this in a separate change if needed.